docs/concepts/storage/persistent-volumes.md

```diff
@@ -417,6 +429,86 @@ spec:
 
 `PersistentVolumes` binds are exclusive, and since `PersistentVolumeClaims` are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
 
+## Raw Block Volume Support
+
+Static provisioning support for Raw Block Volumes is included as an alpha feature for v1.9. With this change are some new API labels that need to be used to facilitate this functionality. Currently, Fibre Channel is the only supported plugin for this feature.
```

Comment:
Please don't refer to the new resource field(s) as label(s). It is confusing because ”label“ in Kubernetes is a special term.

Reply:
Ah, yes. I can see how that can be confusing. I'll update. Thanks, @tengqm